servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -110,8 +110,11 @@ quarkus.resteasy.gzip.enabled=true
 #quarkus.oidc.auth-server-url=
 # fixed at buildtime
 quarkus.http.auth.basic=false
-quarkus.oidc.enabled=false
-
+quarkus.oidc.enabled=true
+quarkus.oidc.client-id=projectnessie
```

Comment:
Is this parameter mandatory if `quarkus.oidc.enabled=true` is set?

Reply:
No. Removed in 089e6c36d10a943566eff5e094a161b4653da443